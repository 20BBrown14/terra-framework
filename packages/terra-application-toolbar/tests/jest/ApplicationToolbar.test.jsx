import React from 'react';

import ApplicationToolbar from '../../src/ApplicationToolbar';

describe('ApplicationToolbar', () => {
  it('should render without optional props', () => {
    const result = shallow((
      <ApplicationToolbar />
    ));
    expect(result).toMatchSnapshot();
  });

  it('should render with provided logo and utility props', () => {
    const result = shallow((
      <ApplicationToolbar
        logo={<ApplicationToolbar.Logo />}
        utility={<ApplicationToolbar.Utility />}
      />
    ));

    expect(result).toMatchSnapshot();
  });
});
