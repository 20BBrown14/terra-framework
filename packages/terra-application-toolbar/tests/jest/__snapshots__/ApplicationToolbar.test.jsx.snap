// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationToolbar should render with provided logo and utility props 1`] = `
<div>
  <div
    className="toolbar-splash"
  />
  <div
    className="toolbar"
  >
    <div
      className="toolbar-body"
    >
      <div
        className="start"
      >
        <Logo
          size="tiny"
        />
      </div>
      <div
        className="end"
      >
        <Utility
          size="tiny"
        />
      </div>
    </div>
  </div>
</div>
`;

exports[`ApplicationToolbar should render without optional props 1`] = `
<div>
  <div
    className="toolbar-splash"
  />
  <div
    className="toolbar"
  />
</div>
`;
